shared/common-adapters/check-circle.tsx

```diff
@@ -10,21 +11,37 @@ type Props = {
   className?: string
   disabled?: boolean
   fontSize?: number
+  selectedColor?: Color
   style?: Styles.StylesCrossPlatform
 }
 
-const CheckCircle = (props: Props) => (
-  <Kb.Icon
-    type={props.checked ? 'iconfont-success' : 'iconfont-circle'}
-    color={
-      !props.disabled && (props.checked || Styles.isMobile)
-        ? Styles.globalColors.blue
-        : Styles.globalColors.black_20
+const CheckCircle = (props: Props) => {
+  const [checked, setChecked] = React.useState(props.checked)
```

Comment:
I don't think this component needs to have this state.

Reply:
I don't want to wait for the data to propagate through the action loop before the box is checked. Is that not actually worth worrying about?

Reply:
No it'll be fast.

Reply:
Well, depending on the caller. But that's up to the caller to manage a waiting state if they're going to call an RPC or something.